### DISCUSSION TITLE
Where is org.forgerock.util.encode.Base64 class located?

### DISCUSSION THREAD

**Discussion by ayush146235** [OpenIdentityPlatform/OpenAM#804] [Q&A] [answered]
The openam-shared module refers to org.forgerock.util.encode.Base64 class (https://github.com/OpenIdentityPlatform/OpenAM/blob/master/openam-shared/src/main/java/com/sun/identity/shared/encode/Base64.java). I am getting ClassDefNotFound error for org.forgerock.util.encode.Base64.

Please help in finding org.forgerock.util.encode.Base64 class

**Comment by maximthomas** [comment, marked as answer]
See 
https://github.com/OpenIdentityPlatform/commons/blob/master/commons/util/util/src/main/java/org/forgerock/util/encode/Base64.java
Maven artifact
 https://mvnrepository.com/artifact/org.openidentityplatform.commons/util
